jina/serve/runtimes/servers/http.py

```diff
@@ -254,3 +260,40 @@ def app(self):
             cors=self.cors,
             logger=self.logger,
         )
+
+
+class SagemakerHTTPServer(FastAPIBaseServer):
+    """
+    :class:`SagemakerHTTPServer` is a FastAPIBaseServer that uses a custom FastAPI app for sagemaker endpoints
+
+    """
+
+    @property
```

Comment:
shoulr we accept the args? for better testability? or is always like this?

Reply:
[It has to be 8080 always](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html#your-algorithms-inference-code-run-image:~:text=the%20container%20must%20have%20a%20web%20server%20listening%20on%20port%208080%20and%20must%20accept%20POST%20requests%20to%20the%20/invocations%20and%20/ping%20endpoints.).